components/form.css

```diff
@@ -0,0 +1,82 @@
+.form{
+    width: 100%;
+}
+
+.label{
+    font-size: 1.25rem;
+    letter-spacing: initial;
+}
+
+.input{
+    margin: 10px auto;
+    border-radius: 4px;
+    outline: none;
+    border: 1px solid var(--gray-700);
+    padding: 10px 1rem;
+    width: 100%;
+    font-size: 14px;
+}
+
+.input_container{
+    margin-bottom: 2rem;
+}
+
+.field_required::after{
+    content: "*";
+    color: var(--red-400);
+}
+
+.form input:focus {
+	outline: 0;
+	border-color: var(--gray-100);
+}
+
+.input_container select{
```

Comment:
don't target element, as the user will have hard time over-riding this.

Reply:
Done!